fbw-a32nx/src/systems/tcas/src/components/TcasSoundManager.ts

```diff
@@ -19,7 +17,7 @@ class PeriodicSound {
   }
 }
 
-export class TcasSoundManager implements TcasComponent {
+export class TcasSoundManager {
```

Comment:
Shouldn't this constructor be private as well as it also looks like a Singleton?

Reply:
Heh, good catch, I didn't notice that one is also like that.